openstack/cronus/templates/nebula/_config.yaml.tpl

```diff
@@ -180,6 +180,16 @@ nebula:
   {{- end }}
 {{- end }}
 {{- end }}
-
+{{- if .Values.nebula.cors.enabled }}
+  cors:
+    enabled: {{ .Values.nebula.cors.enabled }}
+    allowedOrigins:
+{{- range $k, $v := .Values.nebula.cors.allowedOrigins }}
```

Comment:
Just a side note: I noticed that we do this in a few places, but it would potentially be easier to just do:
```
{{- .Values.nebula.cors.allowedOrigins | toYaml | indent 6 }}
```

Reply:
made a mistake in my comment... it should be nindent and not indent